update.sh

```diff
@@ -13,9 +13,11 @@ versions=( "${versions[@]%/}" )
 travisEnv=
 for version in "${versions[@]}"; do
 	latest=$(curl -sSL 'https://owncloud.org/changelog/' |tac|tac| grep -o -m 1 "\(Version\|Release\) ${version}.[[:digit:]]\+" | sed -rn 's/(Version|Release) (.*)/\2/p')
-	sed -ri -e 's/^(ENV OWNCLOUD_VERSION) .*/\1 '"$latest"'/' "$version/Dockerfile"
-	
-	travisEnv='\n  - VERSION='"$version$travisEnv"
+	travisEnv='\n  - VERSION='"$version"' VARIANT='"$travisEnv"
```

Comment:
I thought it was cool to have the `owncloud` image default to apache or `owncloud:8`. But as you want, I can delete it.

[Wordpress](https://hub.docker.com/_/wordpress/) is doing it this way, that's why I wanted to stay consistent.